CMakeLists.txt

```
@@ -3,7 +3,7 @@
 
 cmake_minimum_required(VERSION 3.16)
 
-project(iDynTree VERSION 11.0.0
+project(iDynTree VERSION 11.1.0
```

```suggestion
project(iDynTree VERSION 12.0.0
```

in the meanwhile the changes become ABI breaking (i.e. add new virtual functions to a base class), so let's bump indeed the major version.